src/api/onRequest.ts

```diff
@@ -1,40 +0,0 @@
-import { AxiosError, AxiosRequestConfig } from "axios";
-import { LocalStorageKeys } from "constants/constants";
-
-// Add a request interceptor
-export function onRequest(config: AxiosRequestConfig) {
-  const address: any = localStorage.getItem(LocalStorageKeys.authToken);
-  const TokenAuth = 'Bearer ' + address;
-
-  if (address) {
-    config.headers = {
-      ...config.headers,
-      Authorization: TokenAuth,
-    };
-  }
-
-  return config;
-}
-
-export function onRequestError(error: AxiosError) {
-  if (localStorage.getItem(LocalStorageKeys.isLoggedIn) !== "true") {
-    localStorage.clear();
-    window.location.reload();
-  }
-  return Promise.reject(error);
-}
```

Comment:
Also, this - API error handling ,, Or will we handle this in https://input-output.atlassian.net/browse/PLT-8283

Reply:
Yes, we should improve all the error handling in PLT-8283